### PR TITLE
Resolves #1867 Adding correct status handling BULK MEMO from list

### LINKED ISSUE #1867: BULK MEMO FROM LIST, stats counter only counting as one
@CDPotter, I was looking over your script to update the script list w/ time savings, and it looks like the stats counter is only counting the usage of the script as one instance, instead of it counting each instance within the script. Can you take a look at this one?

### PR DESCRIPTION
**Pull request by CDPotter**
BLIP: Corrected time savings counted to correctly reflect time savings from script.

### CODE REVIEW

**Comment by theVKC** [comment]
@CDPotter: can you tell me why this is three separate pull requests? I'd do separate commits, one pull request (doing three pull requests makes things more challenging on my end to catalog and process).

I won't make you redo it. Just curious is all.

**Comment by CDPotter** [comment]
Figured since they were all separate scripts I'd do them separately in case any issues arose and we needed to backtrack.

**Comment by theVKC** [comment]
The way we'd normally do that is separate commits, but one pull request. I can roll back separate commits as easily as pull requests... separate pull requests just means I have more to track on release notes and internal tracking. Anything we can do to save me some time is very much appreciated. :P

File: Script Files/BULK/BULK - MEMO FROM LIST.vbs [hunk 1]
```diff
@@ -334,4 +335,5 @@ IF privileged_array <> "" THEN
 	MsgBox "The script could not generate a memo for the following cases..." & vbCr & privileged_array
 END IF
 
+STATS_counter = STATS_county - 1  'subtracts one from the stats (since 1 was the count, -1 so it's accurate)
```
Comment (theVKC) [inline Script Files/BULK/BULK - MEMO FROM LIST.vbs]: Same goes here, @CDPotter: `STATS_county` should be `STATS_counter`.

**Pull request merged**